common/subset-tests.js

```diff
@@ -9,13 +9,30 @@
   var subTestEnd = Infinity;
   var match;
   if (location.search) {
-      match = /(?:^\?|&)(\d+)-(\d+|last)(?:&|$)/.exec(location.search);
-      if (match) {
-        subTestStart = parseInt(match[1], 10);
-        if (match[2] !== "last") {
-            subTestEnd = parseInt(match[2], 10);
-        }
+    match = /(?:^\?|&)(\d+)-(\d+|last)(?:&|$)/.exec(location.search);
+    if (match) {
+      subTestStart = parseInt(match[1], 10);
+      if (match[2] !== "last") {
+          subTestEnd = parseInt(match[2], 10);
       }
+    }
+    match = /(?:^\?|&)split=(\d+)(?:&|$)/.exec(location.search);
+    if (match) {
+      var testsPerVariant = parseInt(match[1], 10);
+      add_completion_callback(tests => {
+        var total = tests.length;
+        var template = '<meta name="variant" content="?%s-%s">';
```

Comment:
This looks like temporary code used to help generate the patch. Remove it?

(Or I'm really missing the magic here.)

Reply:
I thought I'd keep it in as utility for the future.

Reply:
Okay - add comments then please?

At first glance I thought it was going to new functionality to do this automagically like `<meta name="split" content="200">` or something and was very confused...

Reply:
Sure.

Reply:
Done.